src/Sulu/Bundle/MediaBundle/Resources/js/views/MediaDetail/README.md

```diff
@@ -0,0 +1,2 @@
+The `MediaDetail` view is registered with the key `sulu_media.detail`. It show a thumbnail of the recent version of the
+uploaded media. Besides that a form is rendered to edit information about the media.
```

Comment:
We actually also list all the route options used in this component with a short explanation. That would be `backRoute` and `locales` in this case. Whereby I am wondering if the `backRoute` must be configureable at all?